pkg/model/openstackmodel/servergroup.go

```diff
@@ -148,7 +164,7 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 			Tags: []string{
 				fmt.Sprintf("%s=%s", openstack.TagKopsInstanceGroup, groupName),
 				fmt.Sprintf("%s=%s", openstack.TagKopsName, portTagKopsName),
-				fmt.Sprintf("%s=%s", openstack.TagClusterName, b.ClusterName()),
+				fmt.Sprintf("%s=%s", openstack.TagClusterName, TruncateString(b.ClusterName(), 42)),
```

Comment:
This would be a magic integer IMO.  I think what you mean is something like `MAX_TAG_LENGTH_OPENSTACK - len(openstack.TagClusterName) + 1`  +1 for the `=` here, which just happens to be 42 however would impact anyone much more than necessary if they changed the value of the constant `openstack.TagClusterName`.